<?xml version="1.0"?>
<launch>
    <node pkg="tensorflow_object_detection" type="object_detection_ros.py" name="object_detection" >
        <param name="per" type="double" value="0.3" />
        <param name="dev" type="string" value="1" />
        <remap from="/image"                  to="/usb_cam/image_raw" />
        <remap from="/object_detection/image" to="/object_detection/image"/>
    </node>
</launch>
